### DISCUSSION TITLE
How to access the test helpers in `common_test.go`?

### DISCUSSION THREAD

**Discussion by DennisTheMenace780** [go-git/go-git#861] [Q&A]
I'm working a small Git utility for myself that is paired with the Charm library and i'd like to create some unit tests. It seems that there are quite a lot of helper functions in the `common_test.go` file that could be really helpful for me, but i'm not able to access them for some reason. 

My end goal here is to build dummy repositories just for my tests. 

Would love some suggestions so I am not just duplicating code.

**Comment by pjbgf** [comment]
Hey Dennis, that's a good point.

I think it will largely depends on the type of tests you have in mind. But it may be worth checking whether we could extract some of the testing code from the `go-git` repository into a new repository within the `go-git` org that could be reusable by `go-git` and other projects.
